model/schema/crdch_model.yaml

```diff
@@ -9769,23 +9769,23 @@ enums:
     comments:
     - 'Name according to TCCM: "CRDC-H.EnvironmentalExposureObservation.method_type"'
     code_set: https://terminology.ccdh.io/enumerations/CRDC-H.EnvironmentalExposureObservation.method_type
-    code_set_version: '2021-09-17T21:00:16.680627+00:00'
+    code_set_version: '2021-09-20T20:21:30.362736+00:00'
```

Comment:
is this a real version? makes it harder to evaluate a PR

Reply:
It is not -- I wanted to have some way of recording when I queried the CCDH Terminology Service, since its content may change between CRDC-H model releases. It would be much better to get some sort of code_set_version from them -- I've created an issue to ask them to include that in their API output: https://github.com/cancerDHC/ccdh-terminology-service/issues/81